internal/provider/resource_rediscloud_active_active_subscription_database.go

```diff
@@ -463,76 +472,88 @@ func resourceRedisCloudActiveActiveSubscriptionDatabaseUpdate(ctx context.Contex
 	subscriptionMutex.Lock(subId)
 	defer subscriptionMutex.Unlock(subId)
 
-	var global_alerts []*databases.UpdateAlert
+	var globalAlerts []*databases.UpdateAlert
 	for _, alert := range d.Get("global_alert").(*schema.Set).List() {
 		dbAlert := alert.(map[string]interface{})
 
-		global_alerts = append(global_alerts, &databases.UpdateAlert{
+		globalAlerts = append(globalAlerts, &databases.UpdateAlert{
 			Name:  redis.String(dbAlert["name"].(string)),
 			Value: redis.Int(dbAlert["value"].(int)),
 		})
 	}
 
+	globalSourceIps := setToStringSlice(d.Get("global_source_ips").(*schema.Set))
+
 	// Make a list of region-specific configurations
+	// TODO: need to set the region not in the override list to the global values and remove the override from the state
```

Comment:
Is this just needing more time or is there something blocking this?

Reply:
Sorry I should have removed that comment, that should be the current behaviour now